doc/customized+model.md

```diff
@@ -0,0 +1,130 @@
+# Customized Model
+
+SQLFlow supports training and predicting using customized models. This documentation explains the design choice and provides a concrete example of adding a new model.
+
+## Keras over Estimator
+
+We choose Keras over Estimator for the following reasons:
+
+1. TensorFlow 2.x will closely integrate with Keras. [ref](https://www.youtube.com/watch?v=k5c-vg4rjBw)
+
+2. Keras provides more documentation in writing customized models than estimators. For customized estimators, I've only found two examples:
+
+   1. `DNNClassifier` that uses `core_layers.Dense`. [ref](<https://github.com/tensorflow/estimator/blob/master/tensorflow_estimator/python/estimator/canned/dnn.py#L200-L226>)
+   2. `DNNClassifier` that uses `tf.layers.dense`. [ref](https://github.com/tensorflow/models/blob/master/samples/core/get_started/custom_estimator.py#L29)
+
+   None of these is suitable for long term development.
+
+3. There are plenty off-the-shelf model repositories written in Keras. To name a few: [ResNet](https://github.com/raghakot/keras-resnet), [Transformer](https://github.com/Lsdefine/attention-is-all-you-need-keras), [Mask  R-CNN](https://github.com/matterport/Mask_RCNN).
```

Comment:
Each of the three URLs doesn't point to a model repository, but a model.

Reply:
Sure, I have changed "model repository" => "model".